sos/plugins/ipa.py

```diff
@@ -108,6 +108,7 @@ def setup(self):
             "/var/log/krb5kdc.log",
             "/var/log/dirsrv/slapd-*/logs/access",
             "/var/log/dirsrv/slapd-*/logs/errors",
+            "/var/log/httpd/error_log",
```

Comment:
This is already collected in the `apache` plugin - if IPA is using it for API access then the best option is probably to add `apache` to the existing `identity` profile.